src/Services/SchemaPoster.php

```diff
@@ -66,48 +70,45 @@ public function __construct(LoggerChannelFactoryInterface $loggerChannelFactory,
    * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
    * @throws \Drupal\search_api\SearchApiException
    * @throws \Drupal\search_api_solr\SearchApiSolrException
-   * @throws \Psr\Http\Client\ClientExceptionInterface
    */
   public function postSchema(string $server_id): array {
     $files = $this->getSolrFiles($server_id);
-    $toReturn = [];
+    $output = [];
     foreach ($files as $filename => $file_contents) {
       try {
         $response = $this->uploadSchemaFile($filename, $file_contents);
-        $logFunction = in_array($response->getStatusCode(), [200, 201, 202, 203]) ? 'notice' : 'error';
         $message = vsprintf('File: %s, Status code: %d - %s', [
           'filename' => $filename,
           'status_code' => $response->getStatusCode(),
           'reason' => $response->getReasonPhrase(),
         ]);
-        $toReturn[] = $message;
-        $this->getLogger()->{$logFunction}($message);
+        $output[] = $message;
+        $this->logger->debug($message);
```

Comment:
Seems like "'File: %s, Status code: %d - %s'" is a debug, so logging this as a debug log